.erb-lint.yml

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
Have added `erb-lint` to project as we have a lot of old erb files which requires some linting. I have fixed files inside `app/views/parsers/`. Once we fix all of them we can add it to CI.